### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by c0b41** [faastjs/faast.js#53]
seems like process-doctor not support win.
```
info "fsevents@1.2.9" is an optional dependency and failed compatibility check. Excluding it from installation.
error process-doctor@1.0.0: The platform "win32" is incompatible with this module.
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

**Comment by acchou** [comment]
Did you try this in wsl bash? I have tested faast.js on Windows in wsl bash and it seemed to work.

**Comment by c0b41** [comment]
@acchou yeah wsl working, but i don't use wsl, maybe fastjs support native windows support?

**Comment by acchou** [comment]
I'll take a closer look at this over the next day or two. It might be possible.

**Comment by c0b41** [comment]
@acchou https://github.com/zeit/fun working great maybe help you..

**Comment by acchou** [comment]
I looked into this and it's certainly possible to eliminate process-doctor, but I don't know if it's worth doing. When using the local provider, part of the point is to allow for local testing of a cloud function. Cloud functions on AWS Lambda and Google Cloud Functions are all linux based, so running locally in non-wsl Windows will not be particularly good for testing... non-native packages will not work, and process cpu measurements won't work without significant effort. As faast.js gets more functionality around things like tail latency optimization, we will increasingly need access to unix apis to access more detailed process metrics.

In short, cloud functions on AWS and Google run Linux; using Windows is fine with WSL. Running faast.js on non-WSL Windows is a pretty unusual corner case, in which case you might be better off simply using node.js Workers or child processes.

**Comment by c0b41** [comment]
@acchou thnx
